spec/controllers/openid_connect/token_controller_spec.rb

```diff
@@ -62,6 +62,11 @@
           })
         action
       end
+
+      it 'does not set session cookie in response' do
+        action
+        expect(response.headers['Set-Cookie']).to_not include(APPLICATION_SESSION_COOKIE_KEY)
```

Comment:
This spec seems to pass on `main`. Is it sufficient?

Reply:
It's difficult to get it to happen in controller tests, not sure why. I added a request spec that does reproduce the issue.